samples/excel/85-preview-apis/worksheet-find-all.yaml

```diff
@@ -0,0 +1,154 @@
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const foundRanges = sheet.findAllOrNullObject("Delay", {
+                    completeMatch: false,
+                    matchCase: false
+                });
+
+                await context.sync();
+
+                if (foundRanges.isNullObject) {
+                    console.log("No delayed projects");
+                } else {
+                    foundRanges.format.fill.color = "yellow"
+                }
+            });
+        }
+
+        async function findCanceled() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                // NOTE: In this sample, there are no canceled projects. Using findAll (without the OrNullObject) would throw an ItemNotFound error.
+                const foundRanges = sheet.findAllOrNullObject("canceled", {
+                    completeMatch: false,
+                    matchCase: false
+                });
+
+                await context.sync();
+
+                if (foundRanges.isNullObject) {
+                    console.log("No canceled projects");
+                } else {
+                    foundRanges.format.fill.color = "red"
+                }
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Sample");
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                const projectTable = sheet.tables.add("A1:D1", true);
+                projectTable.name = "ProjectTable";
+                projectTable.getHeaderRowRange().values = [["Project", "Alpha", "Beta", "Ship"]];
+                projectTable.rows.add(null, [
+                    ["Project 1", "Complete", "Ongoing", "On Schedule"],
+                    ["Project 2", "Complete", "Complete", "On Schedule"],
+                    ["Project 3", "Ongoing", "Not Started", "Delayed"],
+                    ["Project 4", "Complete", "Complete", "On Schedule"],
+                    ["Project 5", "Incomplete", "Delayed", "Delays Likely"]
+                ]);
+
+                sheet.getUsedRange().format.autofitColumns();
+                sheet.activate();
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to find strings across an entire worksheet.</p>
```

Comment:
For consistency with snippet title/description (and with the other 'find' snippet you created previously) -- change "how to find strings..." -> "how to find data..."